tmp/build/patch-charts.sh

```diff
@@ -250,6 +250,22 @@ function patchTemplates() {
     sed -i -e 's/\/health/\/tmp\/health/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
   fi
 
+  # add oauth-proxy default hub, image, tag and imagePullPolicy
+  if [[ "${COMMUNITY,,}" == "true" ]]; then
+    sed -i -e '/^global:/a \
+\  oauthproxy:\
+\    hub: docker.io/openshift\
+\    image: oauth-proxy\
+\    tag: latest\
+\    imagePullPolicy: IfNotPresent\n' ${HELM_DIR}/istio/values.yaml
```

Comment:
Istio upstream has this rule: For `latest` tag, policy is `Always`; for other tags, it's `IfNotPresent`. Not sure if we want to follow them.

Reply:
We should not be using the latest tag in any case, I've requested changes to that part